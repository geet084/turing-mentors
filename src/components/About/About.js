import React, { Component } from 'react';

export class About extends Component {
  render() {
    return (
      <div>
        <h2>About page</h2>
        <p>Turing mentors is about...</p>
      </div>
    )
  }
}

export default About;