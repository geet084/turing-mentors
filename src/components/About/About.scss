.link-content {
  font-size: 1rem;
  padding: 5px;
  color: #888;
}