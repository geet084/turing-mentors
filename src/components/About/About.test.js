import React from 'react';
import { shallow } from 'enzyme';
import { About } from './About';

describe('About', () => {
  let wrapper;
  it('should have match the correct snapshot', () => {
    wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});