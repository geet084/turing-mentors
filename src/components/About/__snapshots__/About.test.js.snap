// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should have match the correct snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          Turing's mission is to unlock human potential by training a diverse and inclusive student body to succeed in high-fulfillment technical careers. We are a community committed to challenging ourselves and each other to build the knowledge, skills, and understanding needed for long-term success.
        </p>,
        <p>
          In addition to that committment is the willingness to help others who are walking in your footsteps, struggling in the same ways you may have struggled.
        </p>,
        <h5>
          Turing Mentors offers a way to connect those on both sides of the Turing program to the betterment of all.
        </h5>,
      ],
      "className": "link-content",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Turing's mission is to unlock human potential by training a diverse and inclusive student body to succeed in high-fulfillment technical careers. We are a community committed to challenging ourselves and each other to build the knowledge, skills, and understanding needed for long-term success.",
        },
        "ref": null,
        "rendered": "Turing's mission is to unlock human potential by training a diverse and inclusive student body to succeed in high-fulfillment technical careers. We are a community committed to challenging ourselves and each other to build the knowledge, skills, and understanding needed for long-term success.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "In addition to that committment is the willingness to help others who are walking in your footsteps, struggling in the same ways you may have struggled.",
        },
        "ref": null,
        "rendered": "In addition to that committment is the willingness to help others who are walking in your footsteps, struggling in the same ways you may have struggled.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Turing Mentors offers a way to connect those on both sides of the Turing program to the betterment of all.",
        },
        "ref": null,
        "rendered": "Turing Mentors offers a way to connect those on both sides of the Turing program to the betterment of all.",
        "type": "h5",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            Turing's mission is to unlock human potential by training a diverse and inclusive student body to succeed in high-fulfillment technical careers. We are a community committed to challenging ourselves and each other to build the knowledge, skills, and understanding needed for long-term success.
          </p>,
          <p>
            In addition to that committment is the willingness to help others who are walking in your footsteps, struggling in the same ways you may have struggled.
          </p>,
          <h5>
            Turing Mentors offers a way to connect those on both sides of the Turing program to the betterment of all.
          </h5>,
        ],
        "className": "link-content",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Turing's mission is to unlock human potential by training a diverse and inclusive student body to succeed in high-fulfillment technical careers. We are a community committed to challenging ourselves and each other to build the knowledge, skills, and understanding needed for long-term success.",
          },
          "ref": null,
          "rendered": "Turing's mission is to unlock human potential by training a diverse and inclusive student body to succeed in high-fulfillment technical careers. We are a community committed to challenging ourselves and each other to build the knowledge, skills, and understanding needed for long-term success.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "In addition to that committment is the willingness to help others who are walking in your footsteps, struggling in the same ways you may have struggled.",
          },
          "ref": null,
          "rendered": "In addition to that committment is the willingness to help others who are walking in your footsteps, struggling in the same ways you may have struggled.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Turing Mentors offers a way to connect those on both sides of the Turing program to the betterment of all.",
          },
          "ref": null,
          "rendered": "Turing Mentors offers a way to connect those on both sides of the Turing program to the betterment of all.",
          "type": "h5",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
